src/Steam.package/STExampleArticle.class/instance/date.st

```diff
@@ -0,0 +1,3 @@
+accessing
+date
+	date ifNil: [ ^ '' ] ifNotNil: [ ^ date asDate]
```

Comment:
Oups, this is not right. Either a method should return always a date or always a string. Returning sometimes one type and sometimes another could lead to problems... For example, the users of this method will start doing "if date then, if string then..."

Regarding this, I'd also avoid plain string, except when what we need are for real strings. So strings are ok for names, but let's use dates for dates.

This maybe means that we have to decide
 - that your articles need a default value for dates
 - or that the date should be mandatory

Reply:
fixed